source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRegistration.h

```diff
@@ -19,4 +20,5 @@
 
 - (void) setActionRenderer:(ACRBaseActionElementRenderer *)renderer cardElementType:(NSNumber *)cardElementType;
 
+- (void) setBaseCardElementRenderer:(ACRBaseCardElementRenderer *)renderer CardElementType:(ACRCardElementType)cardElementType;
```

Comment:
>CardElementType [](start = 75, length = 15)

lowercase: `cardElementType`

Reply:
thanks

---
In reply to: [171734434](https://github.com/Microsoft/AdaptiveCards/pull/1164#discussion_r171734434) [](ancestors = 171734434)